demos/interactive_face_detection_demo/cpp/main.cpp

```diff
@@ -2,228 +2,316 @@
+DEFINE_bool(h, false, h_msg);
+
+constexpr char m_msg[] = "path to an .xml file with a trained Face Detection model";
+DEFINE_string(m, "", m_msg);
+
+constexpr char i_msg[] = "an input to process. The input must be a single image, a folder of images, video file or camera id. Default is 0";
+DEFINE_string(i, "0", i_msg);
+
+constexpr char bb_enlarge_coef_msg[] = "coefficient to enlarge/reduce the size of the bounding box around the detected face. Default is 1.2";
+DEFINE_double(bb_enlarge_coef, 1.2, bb_enlarge_coef_msg);
+
+constexpr char d_msg[] =
+    "specify a device to infer on (the list of available devices is shown below). "
+    "Use '-d HETERO:<comma-separated_devices_list>' format to specify HETERO plugin. "
+    "Use '-d MULTI:<comma-separated_devices_list>' format to specify MULTI plugin. "
+    "Default is CPU";
+DEFINE_string(d, "CPU", d_msg);
+
+constexpr char dx_coef_msg[] = "coefficient to shift the bounding box around the detected face along the Ox axis";
+DEFINE_double(dx_coef, 1, dx_coef_msg);
+
+constexpr char dy_coef_msg[] = "coefficient to shift the bounding box around the detected face along the Oy axis";
+DEFINE_double(dy_coef, 1, dy_coef_msg);
+
+constexpr char fps_msg[] = "maximum FPS for playing video";
+DEFINE_double(fps, -std::numeric_limits<double>::infinity(), fps_msg);
+
+constexpr char lim_msg[] = "number of frames to store in output. If 0 is set, all frames are stored. Default is 1000";
+DEFINE_uint32(lim, 1000, lim_msg);
+
+constexpr char loop_msg[] = "enable reading the input in a loop";
+DEFINE_bool(loop, false, loop_msg);
+
+constexpr char mag_msg[] = "path to an .xml file with a trained Age/Gender Recognition model";
+DEFINE_string(mag, "", mag_msg);
+
+constexpr char mam_msg[] = "path to an .xml file with a trained Antispoofing Classification model";
+DEFINE_string(mam, "", mam_msg);
+
+constexpr char mem_msg[] = "path to an .xml file with a trained Emotions Recognition model";
+DEFINE_string(mem, "", mem_msg);
+
+constexpr char mhp_msg[] = "path to an .xml file with a trained Head Pose Estimation model";
+DEFINE_string(mhp, "", mhp_msg);
+
+constexpr char mlm_msg[] = "path to an .xml file with a trained Facial Landmarks Estimation model";
+DEFINE_string(mlm, "", mlm_msg);
+
+constexpr char o_msg[] = "name of the output file(s) to save";
+DEFINE_string(o, "", o_msg);
+
+constexpr char r_msg[] = "output inference results as raw values";
+DEFINE_bool(r, false, r_msg);
+
+constexpr char show_msg[] = "(don't) show output";
+DEFINE_bool(show, false, show_msg);
+
+constexpr char show_emotion_bar_msg[] = "(don't) show emotion bar";
+DEFINE_bool(show_emotion_bar, false, show_emotion_bar_msg);
+
+constexpr char smooth_msg[] = "(don't) smooth person attributes";
+DEFINE_bool(smooth, false, smooth_msg);
+
+constexpr char t_msg[] = "probability threshold for detections. Default is 0.5";
+DEFINE_double(t, 0.5, t_msg);
+
+constexpr char u_msg[] = "resource utilization graphs: -u cdm. "
```

Comment:
```suggestion
constexpr char u_msg[] = "resource utilization graphs. Default is cdm. "
```
Yet another improvement :)